jabichris/Politico-v1
# #163934937 login a user

#### What does this PR do?
This PR will help the user to login in his/her account.

#### Description of Task to be completed?
I want to give users access to the platform based on their respective types

#### How should this be manually tested?
After this branch being merged on develop branch and cloned [git@github.com](mailto:git@github.com):jabichris/Politico-v1.git . Then you should run **npm install** for installing all dependencies, and this script **(npm run db)** for the creation of database tables. Then run **npm run dev** so that the server can be started. After in the postgres you put payloads and POST it on Url:http://localhost:3000/api/v1/auth/login.

#### What are the relevant pivotal tracker stories?
#163934937
#### Screenshots (if appropriate)
![screen shot 2019-02-20 at 11 58 08](https://user-images.githubusercontent.com/33568373/53084785-3401f780-350a-11e9-8576-1b1785ed5634.png)
